main.go

```diff
@@ -64,6 +64,8 @@ func main() {
 		sshFlags     = flag.String("ssh-flags", "", "custom SSH flags")
 		syncBack     = flag.Bool("b", false, "sync extensions back on termination")
 		printVersion = flag.Bool("version", false, "print version information and exit")
+		port         = flag.String("port", "", "Start VS Code on the provided port. If one is not provided a random one is selected")
+		noOpen       = flag.Bool("no-open", false, "Start VS Code but don't open a Chrome app window")
```

Comment:
lets use $DISPLAY = "" for this

Reply:
Is `DISPLAY` set on all macs?

Reply:
Yes, similar to how there is an X11 socket whose path is given by $DISPLAY, $DISPLAY on macOS contains the quartz socket.

Reply:
What about WSL?

Reply:
WSL is almost certainly using some sort of X11 forwarding so it would have $DISPLAY.